.gitignore

```diff
@@ -15,7 +15,6 @@
 /tmp
 
 config/database.yml
-config/site.yml
```

Comment:
We can only do this, if there is a way to update the app configuration without touching the conf we store in github. Is that already possible?